### PR TITLE
Docs: Basic Components Update

### PR DESCRIPTION
**Pull request by ericnakagawa**
This is an improvement to basic components docs.

* I updated the basic components example code to better render components on iOS (added paddingTop).
* I also modified the code to allow reader to easily copy, paste, and then run the code in their project if they followed the 'Getting Started' quick start guide.
* I also added additional copy to clarify suggested usage/guidelines.

### CODE REVIEW

**Comment by facebook-github-bot-4** [comment]
By analyzing the blame information on this pull request, we identified @JoelMarcey to be a potential reviewer.

File: docs/Basics-Component-ListView.md [hunk 1]
```diff
@@ -9,27 +9,29 @@ next: basics-integration-with-existing-apps
 
 On mobile devices, lists are a core element in many applications. The [`ListView`](/react-native/docs/listview.html#content) component is a special type of [`View`](/react-native/docs/tutorial-component-view.html) that displays a vertically scrolling list of changing data.
 
-The `ListView` component requires two properties, `dataSource` and `renderRow`. `dataSource` is the actual source of information that will be part of the list. `renderRow` takes the data and returns a renderable component to display.
+The `ListView` component requires two properties, `dataSource` and `renderRow`. `dataSource` is the source of information for the list. `renderRow` takes one item from the source source and returns a formatted component to render.
```
Comment (JoelMarcey) [inline docs/Basics-Component-ListView.md]: double "source" words here.

File: docs/Basics-Component-Text.md [hunk 2]
```diff
@@ -12,15 +12,40 @@ The most basic component in React Native is the [`Text`](/react-native/docs/text
 This example displays the `string` `"Hello World!"` on the device.
 
 ```JavaScript
-import React from 'react';
+import React, { Component } from 'react';
```
Comment (JoelMarcey) [inline docs/Basics-Component-Text.md]: Do we need `Component` for this simple example?

**Comment by JoelMarcey** [comment]
@ericnakagawa Looks good! Thanks for the enhancements. Just a couple of comments to look at (and possibly update) and then we can shipit.

**Comment by ericnakagawa** [comment]
@JoelMarcey fixes applied and pushed.

**Comment by JoelMarcey** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-0** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1770587729838682/int_phab) to review.

**Pull request closed without merging**